action.yml

```diff
@@ -0,0 +1,5 @@
+name: 'llvm-cbe docker'
+description: 'Prepares llvm-cbe docke'
+runs:
+  using: 'docker'
+  image: 'Dockerfile'
```

Comment:
I'm assuming this file also needs to be updated to point at
https://github.com/llvm/llvm-project/releases/download/llvmorg-10.0.0/clang+llvm-10.0.0-x86_64-linux-gnu-ubuntu-18.04.tar.xz and do a binary build / test?